userspace/engine/falco_utils.h

```diff
@@ -45,5 +49,11 @@ namespace network
 {
 static const std::string UNIX_SCHEME("unix://");
 bool is_unix_scheme(const std::string& url);
+#if defined(__linux__) and !defined(MINIMAL_BUILD) and !defined(__EMSCRIPTEN__)
+// todo: consider extending libs and expose API for ipv4 and ipv6 to string conversion
+std::string ipv4addr_to_string(uint32_t addr);
+std::string ipv6addr_to_string(const ipv6addr& addr);
```

Comment:
Why not make it a function of the ipv6addr structure ?
Since that structure has a constructor that takes a string as input, a `to_string` method would make sense as well. What do you think ?

Reply:
Thought the same. That's why I left a todo item.
@FedeDP would it be ok to make the to_string functions public? For ipv4 we have it in libs, but it's private. For ipv6 we do not yet have it. I can port this code over to libs if you all agree.

Reply:
+1 from my side! Neat idea!

Reply:
(i don't think it's a problem to make them public!)